### ISSUE TITLE
unneeded postinstall.sh command?

### ISSUE THREAD

**Issue by dgobaud** [ueno-llc/react-native-starter#213]
think below is unneeded - not entirely sure what it was for but I see the file is gone from current RNN code base https://github.com/wix/react-native-navigation/tree/master/lib/android/app/src/main/java/com/reactnativenavigation/react

I'm betting they used to have a common NavigationReactNativeHost.java file and have no specialized it based on the RN version so the `rm` is no longer needed?

1. https://github.com/wix/react-native-navigation/blob/master/lib/android/app/src/reactNative51/java/com/reactnativenavigation/react/NavigationReactNativeHost.java
2. https://github.com/wix/react-native-navigation/blob/master/lib/android/app/src/reactNative55/java/com/reactnativenavigation/react/NavigationReactNativeHost.java
3. https://github.com/wix/react-native-navigation/blob/master/lib/android/app/src/reactNative56/java/com/reactnativenavigation/react/NavigationReactNativeHost.java
4. https://github.com/wix/react-native-navigation/blob/master/lib/android/app/src/reactNative57_5/java/com/reactnativenavigation/react/NavigationReactNativeHost.java

https://github.com/ueno-llc/react-native-starter/blob/b01e8d62ac75c2e748dda709191415ca837e6039/scripts/postinstall.sh#L4

```sh
rm -rf ./node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/react/NavigationReactNativeHost.java
```

**Comment by jeremybarbet** [comment]
You are right, it's a bit of legacy with the first version of React native navigation that we had to patch before.

If I remember correctly, couple of files from the rename script are also undefined because of some legacy, no big deal here, but a clean would be really nice.

Don't hesitate to submit pull request if you already made changes on your side, I or Birkir will review it and merge it

**Comment by dgobaud** [comment]
done. curious what is difference in master and develop branch?
